src/api-reference/request/v4.endpoints.markdown

```diff
@@ -25,33 +25,34 @@ If a new or existing project please use version [3.0](./v3.request.html) of this
 * [Travel Agency Resources](#travel-agency-resource)
   * [Get the description of a travel agency office](#get-travel-agency-description-resource)
 * [Schema](#schema)
-  * [Request](#schema-request)
-  * [ApprovalStatus](#schema-approvalstatus)
-  * [Customfield](#schema-customfield)
-  * [Location](#schema-location)
-  * [Amount](#schema-amount)
-  * [ResourceLink](#schema-resourcelink)
-  * [RequestLink](#schema-requestlink)
-  * [Link](#schema-link)
-  * [Expense](#schema-expense)
-  * [Allocation](#schema-allocation)
-  * [Exchange Rate](#schema-exchangerate)
-  * [Expense Type](#schema-expensetype)
-  * [Travel Allowance](#schema-travelallowance)
-  * [Trip Data](#schema-tripdata)
-  * [Vendor](#schema-vendor)
-  * [Segment Leg](#schema-segmentleg)
-  * [Segment Type](#schema-segmenttype)
+  * [Request](#request)
+  * [ApprovalStatus](#approvalstatus)
+  * [Customfield](#customfield)
+  * [List of Requests](#listofrequests)
+  * [Location](#location)
+  * [Amount](#amount)
+  * [ResourceLink](#resourcelink)
+  * [RequestLink](#requestlink)
+  * [Link](#link)
+  * [Expense](#expense)
+  * [Allocation](#allocation)
+  * [Exchange Rate](#exchangerate)
+  * [Expense Type](#expensetype)
+  * [Travel Allowance](#travelallowance)
+  * [Trip Data](#tripdata)
+  * [Vendor](#vendor)
+  * [Segment Leg](#segmentleg)
+  * [Segment Type](#segmenttype)
 
 ## HTTP Status Codes and Headers
```

Comment:
@laurentlhermine : can you please delete "and Headers" as currently this page is not updated/reviewed